### PR TITLE
Updated Readme to make the badges as flags.

### PR DESCRIPTION
**Pull request by mohammedzee1000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # CentOS Community Container Pipeline
 
-[![Build Status](https://ci.centos.org/view/Container/job/centos-container-pipeline-service-ci-pr/badge/icon)](https://ci.centos.org/view/Container/job/centos-container-pipeline-service-ci-pr/)
+| Last PR Build | [![Build Status](https://ci.centos.org/view/Container/job/centos-container-pipeline-service-ci-pr/badge/icon)](https://ci.centos.org/view/Container/job/centos-container-pipeline-service-ci-pr/) |
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 
 CentOS Community Container Pipeline(cccp) is a process, to provide any opensource developer, a platform for containerising their application(s). This process builds the application(s) from any arbitary git repository/repositories, package the built application along with its runtime in a container image, tests the image with help of test script and delivers to a publicly available registry. A user can anytime pull the tested image from that registry.
```
Comment (navidshaikh) [inline README.md]: CentOS ..(cccp) is a service, [..] Open Source, [..] test scripts and atomic scanners and [..]

Comment (mohammedzee1000) [inline README.md]: Are you telling me to change this as well in the readme?

File: README.md [hunk 2]
```diff
@@ -27,6 +28,8 @@ We want to provide a single input interface to the system (pipeline index) and d
     * Pulp or a registry provided by OpenShift, deployed at https://registry.centos.org/
 5. Failure UI
     * Probably part of Input Interface, presenting logs from failed builds
+6. Scan
+    * Scan uses atomic scan technology, using which atomic scanners are used to scan the images for various checks such as is the image upto date and what packages(from rpm packages, to npm packages and so on) are setup in the container and so on.
```
Comment (navidshaikh) [inline README.md]: Scan uses [atomic scan] (https://github.com/projectatomic/atomic) tooling, multiple atomic scanners are run on built images and different checks are done, checking if image has outdated RPM, npm, pip packages and if image has tampered files present, etc.

**Comment by mohammedzee1000** [comment]
#dotests

**Comment by mohammedzee1000** [comment]
All checks have passed on here :)

**Pull request merged**